gallery/src/components/NxDropdown/NxDropdownPage.tsx

```diff
@@ -258,6 +260,16 @@ const NxDropdownPage = () =>
                 any <NxCode>.nx-dropdown-right-button</NxCode>s.
               </NxTable.Cell>
             </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell><NxCode>nx-dropdown-icon</NxCode></NxTable.Cell>
+              <NxTable.Cell>
+                Set custom icons to the correct color.
+              </NxTable.Cell>
+              <NxTable.Cell>
+                This class should be applied to custom icons that are not colored{' '}
```

Comment:
"are not colored" might not be the clearest phrasing. Maybe "do not have a built-in color"

Reply:
> "are not colored" might not be the clearest phrasing. Maybe "do not have a built-in color"

Updated: 34e67258d5977d2fd3d77619b29f01a416f2bfaf